src/java/za/co/mip/ablduck/ABLDuckClassVisitor.java

```diff
@@ -0,0 +1,214 @@
+/**
+ * Copyright 2017 MIP Holdings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+package za.co.mip.ablduck;
+
+import java.util.ArrayList;
+
+import java.io.File;
+import java.io.IOException;
+
+import com.openedge.core.runtime.IPropath;
+
+import eu.rssw.pct.oedoc.ClassDocumentationVisitor;
+
+import eu.rssw.rcode.ClassCompilationUnit;
+import eu.rssw.rcode.Method;
+import eu.rssw.rcode.Property;
+import eu.rssw.rcode.Parameter;
+import eu.rssw.rcode.AccessModifier;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'eu.rssw.rcode.AccessModifier'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AUselessImportCheck)